Client/Frontend/Browser/Tab.swift

```diff
@@ -64,6 +64,7 @@ class Tab: NSObject {
     var restoring: Bool = false
     var pendingScreenshot = false
     var url: URL?
+    var contentBlocker: AnyObject?
```

Comment:
ah yes. Because the class isnt availible in ios10. I think if you add a comment here about why we can remember to remove it later.